web-ng/src/app/components/map/map.component.ts

```diff
@@ -294,6 +320,30 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     });
   }
 
+  private addPolygonToMap(feature: PolygonFeature) {
+    const vertices: google.maps.LatLng[] = [];
+    feature.polygonVertices.map(vertex => {
+      vertices.push(new google.maps.LatLng(vertex.latitude, vertex.longitude));
+    });
+    // Construct the polygon.
+    const polygon = new google.maps.Polygon({
+      paths: vertices,
+      strokeColor: '#ff9131',
+      strokeOpacity: 1,
+      strokeWeight: 4,
```

Comment:
Should we make it thicker when selected?

Reply:
updated in function
`polygon.addListener('click', () => {
      polygon.setOptions({ strokeWeight: 6 });
      this.panAndZoom(vertices[0]);
      this.zone.run(() => {
        this.navigationService.selectFeature(feature.id);
      });
    });
`